recipes-core/initscripts/files/mdev/ttyUSB.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Look for ttyUSB devices which match the known location of the USB to UART
+# adapter connected to the Cypress Bluetooth on the mangOH Yellow. When such a
+# device is identified, either spawn or kill btattach depending on whether the
+# UART is being added or removed respectively.
+if [ $DEVNAME ] && [ $DEVPATH == "/devices/7c00000.hsic_host/usb1/1-1/1-1.2/1-1.2:1.0/$DEVNAME/tty/$DEVNAME" ]
```

Comment:
Can you please document where/how you generated the "/devices..." DEVPATH uevent from the kernel for our CF3 - is this the same on the WP77 & 76? If later we use WP85 could this be problematic? Also, what the regex for DEVNAME should be.

Reply:
We have explicitly decided that we aren't supporting wp85/75 on yellow.  If that changes, we can re-evaluate this.  I'll provide a comment indicating why the path is the way it is.

Reply:
I verified that the path is the same on wp76 and 77